src/components/forms/SignUpForm.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import userEvent from '@testing-library/user-event';
+import * as useActions from '../../hooks/useActions';
+import SignUpForm from './SignUpForm';
+
+jest.mock('../../store/user/patientOperations', () => ({}));
+
+type Default = string;
```

Comment:
What for do you need type for type string, use string instead.

Reply:
Deleted